src/ARToolkitNFT_node.ts

```diff
@@ -0,0 +1,496 @@
+    ARToolkitNFT.AR_DEFAULT_DEBUG_MODE = instance.AR_DEFAULT_DEBUG_MODE;
+
+    /* for arlabelingThresh */
+    ARToolkitNFT.AR_DEFAULT_LABELING_THRESH =
+      instance.AR_DEFAULT_LABELING_THRESH;
+
+    /* for arImageProcMode */
+    ARToolkitNFT.AR_IMAGE_PROC_FRAME_IMAGE = instance.AR_IMAGE_PROC_FRAME_IMAGE;
+    ARToolkitNFT.AR_IMAGE_PROC_FIELD_IMAGE = instance.AR_IMAGE_PROC_FIELD_IMAGE;
+    ARToolkitNFT.AR_DEFAULT_IMAGE_PROC_MODE =
+      instance.AR_DEFAULT_IMAGE_PROC_MODE;
+
+    /* for arGetTransMat */
+    ARToolkitNFT.AR_MAX_LOOP_COUNT = instance.AR_MAX_LOOP_COUNT;
+    ARToolkitNFT.AR_LOOP_BREAK_THRESH = instance.AR_LOOP_BREAK_THRESH;
+
+    /* Enums */
+    ARToolkitNFT.AR_LOG_LEVEL_DEBUG = instance.AR_LOG_LEVEL_DEBUG;
+    ARToolkitNFT.AR_LOG_LEVEL_INFO = instance.AR_LOG_LEVEL_INFO;
+    ARToolkitNFT.AR_LOG_LEVEL_WARN = instance.AR_LOG_LEVEL_WARN;
+    ARToolkitNFT.AR_LOG_LEVEL_ERROR = instance.AR_LOG_LEVEL_ERROR;
+    ARToolkitNFT.AR_LOG_LEVEL_REL_INFO = instance.AR_LOG_LEVEL_REL_INFO;
+
+    ARToolkitNFT.AR_LABELING_THRESH_MODE_MANUAL =
+      instance.AR_LABELING_THRESH_MODE_MANUAL;
+    ARToolkitNFT.AR_LABELING_THRESH_MODE_AUTO_MEDIAN =
+      instance.AR_LABELING_THRESH_MODE_AUTO_MEDIAN;
+    ARToolkitNFT.AR_LABELING_THRESH_MODE_AUTO_OTSU =
+      instance.AR_LABELING_THRESH_MODE_AUTO_OTSU;
+    ARToolkitNFT.AR_LABELING_THRESH_MODE_AUTO_ADAPTIVE =
+      instance.AR_LABELING_THRESH_MODE_AUTO_ADAPTIVE;
+
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_NONE =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_NONE;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_PATTERN_EXTRACTION =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_PATTERN_EXTRACTION;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_GENERIC =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_GENERIC;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_CONTRAST =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_CONTRAST;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_BARCODE_NOT_FOUND =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_BARCODE_NOT_FOUND;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_BARCODE_EDC_FAIL =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_BARCODE_EDC_FAIL;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_CONFIDENCE =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_MATCH_CONFIDENCE;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_POSE_ERROR =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_POSE_ERROR;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_POSE_ERROR_MULTI =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_POSE_ERROR_MULTI;
+    ARToolkitNFT.AR_MARKER_INFO_CUTOFF_PHASE_HEURISTIC_TROUBLESOME_MATRIX_CODES =
+      instance.AR_MARKER_INFO_CUTOFF_PHASE_HEURISTIC_TROUBLESOME_MATRIX_CODES;
+
+    return this;
+  }
+
+  public setup(width: number, height: number, cameraId: number): number {
+    return this.instance.setup(width, height, cameraId);
+  }
+
+  public teardown(): void {
+    this.instance.teardown();
+  }
+
+  public setupAR2(): void {
```

Comment:
it should accept the id argument, as other method here from the underlying Emscripten generated code.